packages/php-wasm/universal/src/lib/php.ts

```diff
@@ -1364,6 +1364,7 @@ export class PHP implements Disposable {
 			this.setSapiName(this.#sapiName);
 		}
 
+		// TODO: Only copy the /internal directory if it's not a shared filesystem.
```

Comment:
This sounds pretty important before merging

Reply:
This change appears to break runtime rotation. I ended up removing the condition and relying on `copyFS()` to only copy MEMFS nodes. The updated comment is:
```
		// Since the underlying filesystem is not shared, we need to copy
		// the old /internal directory to the new filesystem.
		// This is safe, even when predominantly using NODEFS because
		// copyFS only copies MEMFS nodes.
```

Reply:
I would like to add an even more explicit comment about why it is needed, but I am at the end of the day and don't yet know exactly why it is necessary for runtime rotation to work.